odoo/models.py

```diff
@@ -1972,6 +1985,16 @@ def clear_caches(cls):
         ``tools.ormcache`` or ``tools.ormcache_multi``.
         """
         cls.pool._clear_cache()
+        if (getattr(threading.currentThread(), 'testing', False) or
+            cls.pool.in_test_mode() or config.get('test_enable') or CLEAR_CACHE_VERBOSE):
```

Comment:
Why the code is different to original one here?

I mean, it will show the verbose if the flag is disabled like the opposite focus

Could you check the whole original code, please?

Reply:
Missed a `not` from original patch. Ammended and pushed newest.